lib/language_pack/helpers/bundler_wrapper.rb

```diff
@@ -130,4 +186,27 @@ def parse_gemfile_lock
       Bundler::LockfileParser.new(gemfile_contents)
     end
   end
+
+  def major_bundler_version
+    # https://rubular.com/r/uuRpai9IheL68d
+    bundler_version_match = @gemfile_lock_path.read.match(/^BUNDLED WITH$(\r?\n)   (?<major>\d*)\.\d*\.\d*/m)
```

Comment:
Should these be `+` and not `*`?

Reply:
Also, scraping `BUNDLED_WITH` seems awful :(

Reply:
> Should these be `+`

Great call.

> Also, scraping `BUNDLED_WITH` seems awful :(

Agreed, though It's the least awful thing I could come up with :(